### PR TITLE
Updated world map with real percentages

### LINKED ISSUE #4274: "Where are Ubuntu users" map seems wrong
North America is seemingly low on users, but North America is a very popular location for users.

I wonder if the map is not reflecting the data correctly.  Perhaps the colours are inverted?

---
*Reported from: https://www.ubuntu.com/desktop/statistics*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated world map data with real data on /desktop/statistics
- black and linter updated formatting of related .js files

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/desktop/statistics](http://0.0.0.0:8001/desktop/statistics)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the world data looks more like we think it should

## Screenshots

image

### CODE REVIEW

File: static/js/desktopStatistics.js [hunk 1]
```diff
@@ -81,71 +94,93 @@ function manipulateData(data, options) {
 
 function colorShade(usageRange, colors) {
   var index = 0;
-  if (usageRange <= 1) {
+  if (usageRange <= 0.003326016523) {
```
Comment (anthonydillon) [inline static/js/desktopStatistics.js]: I'm not sure this is right. I changed the take before but was told to use the snapcraft range

Comment (pmahnke) [reply static/js/desktopStatistics.js]: I bet snapcraft has it wrong, I broke it into 5 percentiles, which is a truer visualisation.  I looked at lots of info on this, but I will chat with Juan

**Comment by anasereijo** [comment]
@pmahnke Colours (from lightest to darkest): #FAD8CB #F5BEAB #EEA78F #E79173 #DF6F47 #E95420

File: templates/desktop/statistics.html [hunk 2]
```diff
@@ -72,14 +72,14 @@ <h3>Clean install or upgrade?</h3>
...
+<section class="p-strip">
   <div class="row p-mobile-flex-col u-equal-height u-vertically-center">
-    <div class="col-4">
+    <div class="col-6">
       <h3>Where are Ubuntu users?</h3>
       <p>Location is derived from the user setting the timezone and location in the installer and not from an identifiable IP address.</p>
-    </div>
-    <div class="col-8 u-align--center p-where-user-are">
-      <svg id="where-are-users" height="250" width="500"></svg>
+      <<div style="overflow-x: scroll;">
```
Comment (Caleb-Ellis) [inline templates/desktop/statistics.html]: there's an extra `<` here

**Comment by anthonydillon** [comment]
This seems borked. Superseded by https://github.com/canonical-websites/www.ubuntu.com/pull/4327

**Pull request closed without merging**